content/en/logs/log_collection/javascript.md

```diff
@@ -160,6 +163,75 @@ This gives the following result:
 }  
 ```
 
+## Advanced usage
+
+### Filter by severity
+
+In some cases, you might want to disable the debug mode or to only collect warning and errors. This can be achieved by changing the logging level thanks to the `logLevel` parameter to `debug`, `info`, `warn`, and `error` :
```

Comment:
```suggestion
In some cases, you might want to disable the debug mode or only to collect warning and errors. This can be achieved by changing the logging level thanks to the `logLevel` parameter to `debug`, `info`, `warn`, and `error` :
```

this is called a split infinitive